ci: E2E test failed

# Bug report

https://github.com/apache/apisix-dashboard/runs/1474172932#step:5:86

```
--- FAIL: TestConsumer_with_key_auth (0.43s)
Error:     printer.go:54: PUT http://127.0.0.1:8080/apisix/admin/routes/r1
Error:     printer.go:54: GET http://127.0.0.1:9080/hello
    reporter.go:23: 
        	Error Trace:	reporter.go:23
        	            				chain.go:21
        	            				response.go:585
        	            				response.go:151
        	            				base.go:182
        	            				consumer_test.go:131
        	Error:      	
        	            	expected status equal to:
        	            	 "401 Unauthorized"
        	            	
        	            	but got:
        	            	 "404 Not Found"
        	Test:       	TestConsumer_with_key_auth
Error:     printer.go:54: PUT http://127.0.0.1:8080/apisix/admin/consumers
```

I can not sure if we need to revert this commit: https://github.com/apache/apisix-dashboard/commit/666e8a203d8dcc968b526154a01f6eb40b077103

--- a/api/test/e2e/base.go
+++ b/api/test/e2e/base.go
@@ -115,7 +115,7 @@ func APISIXHTTPSExpect(t *testing.T) *httpexpect.Expect {
 	return e
 }
 
-var sleepTime = time.Duration(100) * time.Millisecond
+var sleepTime = time.Duration(300) * time.Millisecond
 
 type HttpTestCase struct {
 	caseDesc      string

